gcloud/datastore/entity.py

```diff
@@ -104,55 +102,6 @@ def exclude_from_indexes(self):
         """
         return frozenset(self._exclude_from_indexes)
 
-    @property
-    def _must_key(self):
```

Comment:
Entities no longer save themselves, but `_get_dataset_id_from_keys` will blow up during `datastore.put()` if there is no key.